addons/controllers/kapp-controller/suite_test.go

```diff
@@ -0,0 +1,162 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
```

Comment:
Question: Could we just add the kapp-controller reconcile to suite_test.go in addons? That is the actual use case  right? All controllers running in the same binary. If we do it separately we aren't catching any race conditions or unforeseen side effects.

Reply:
Good point!